components/automate-chef-io/content/docs/backup.md

```diff
@@ -131,10 +131,10 @@ Success: Created backup 20180518010336
 
 ## Storing Backups in a Single-file Archive for Airgapped Installations
 
-The backup directory contains both the timestamp-based directory for the configuration, and the reporting data stored in the `automate-elasticsearch-data` directory. 
+The backup directory contains both the timestamp-based directory for the configuration, and the reporting data stored in the `automate-elasticsearch-data` directory.
```

Comment:
```suggestion
The backup directory contains both the timestamp-based directory for the configuration and the reporting data stored in the `automate-elasticsearch-data` directory.
```
I'm not sure that I understand the single-file aspect of this. Is the header correct and relevant?